graylog2-web-interface/src/views/components/visualizations/utils/getFieldNameFromTrace.ts

```diff
@@ -19,6 +19,8 @@ import type Series from 'views/logic/aggregationbuilder/Series';
 import { parseSeries } from 'views/logic/aggregationbuilder/Series';
 
 const getFieldNameFromTrace = ({ series, fullPath }: { series: Array<Series>, fullPath: string }) => {
```

Comment:
We should also update the type of `fullPath`. Currently it reads like it is always a string.